app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details.scala.html

```diff
@@ -144,18 +145,36 @@
     </table>
 
     <div class="grid-row mt-2">
-        <span class="column-one-third bold">@messages("case.activity.assigned-to")</span>
-        <span id="activity-events-assignee" class="column-two-thirds">
-        @{
-            if(c.assignee.map(_.id).contains(request.operator.id)) {
-                "You"
-            } else if(c.assignee.flatMap(_.name).isDefined) {
-                c.assignee.get.name.get
-            } else if(c.assignee.isDefined) {
-                s"PID ${c.assignee.get.id}"
+        <div class="column-full">
+            @if(c.assignee.isDefined) {
+                <span id="activity-events-assignee-label">
+                    @messages("case.activity.assigned-to")
+                </span>
+                <span id="activity-events-assignee">
+                @{
+                    if(c.assignee.map(_.id).contains(request.operator.id)) {
+                        "you"
+                    } else if(c.assignee.flatMap(_.name).isDefined) {
+                        c.assignee.get.name.get
+                    } else if(c.assignee.isDefined) {
+                        s"PID ${c.assignee.get.id}"
+                    }
+                }
+                </span>
             } else {
-                "This case is currently unassigned"
+                <span id="activity-events-assigned-queue-label">
+                    @messages("case.activity.assigned-queue")
+                </span>
+                <span id="activity-events-assigned-queue">
+                @{
+                    if(c.queueId.isDefined) {
+                        Queues.queueNames(c.queueId.get)
+                    } else {
+                        "Gateway"
```

Comment:
Just checking that it's ok to hard-code the Gateway name here.

Reply:
You could use `Queues.gateway.name`